amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1790,6 +1790,33 @@ VX_API_ENTRY vx_node VX_API_CALL vxExtrppNode_ResizeCropMirrorPD(vx_graph graph,
     return node;
 }
 
+VX_API_ENTRY vx_node VX_API_CALL vxExtrppNode_ResizeMirrorNormalizeTensor(vx_graph graph, vx_image pSrc, vx_array srcImgWidth, vx_array srcImgHeight,vx_image pDst,vx_array dstImgWidth,vx_array dstImgHeight,vx_array mean, vx_array std_dev, vx_array flip, vx_scalar chnShift, vx_uint32 nbatchSize)
+{
+    vx_node node = NULL;
+    vx_context context = vxGetContext((vx_reference)graph);
+    if(vxGetStatus((vx_reference)context) == VX_SUCCESS) {
+        vx_uint32 dev_type = getGraphAffinity(graph);
+        vx_scalar DEV_TYPE = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &dev_type);
+        vx_scalar NBATCHSIZE = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &nbatchSize);
+        vx_reference params[] = {
+            (vx_reference) pSrc,
+            (vx_reference) srcImgWidth,
+            (vx_reference) srcImgHeight,
+            (vx_reference) pDst,
+            (vx_reference) dstImgWidth,
+            (vx_reference) dstImgHeight,
+            (vx_reference) mean,
+            (vx_reference) std_dev,
+            (vx_reference) flip,
+            (vx_reference) chnShift,
+            (vx_reference) NBATCHSIZE,
```

Comment:
@SundarRajan28 I think Rajy also mentioned this once. Please do not have capital letters for variables.

Reply:
This has been the convention in the other kernel definitions so if the changes are required, we'll create a new PR for the variable name changes in all the other definitions

Reply:
yeah. I think batch_size needs to be removed as argument in future PR. Once we have a unified RPP handle for all the kernels, batch_size needs to be removed as argument. RPP can use tensor dims to figure out the batch_size